src/resources/shipment.js

```diff
@@ -0,0 +1,158 @@
+import T from 'proptypes';
```

Comment:
$> api.Shipment.all().then(console.log)
Promise { <pending> }
$> TypeError: this.unwrapAll(...).map is not a function

Tracker has a `unwrapAll` fn, this one doesn't. I assume this is missing that function? Insurances, and some other objects should probably have that fn as well.